tools/build-tools/src/fluidBuild/tasks/leaf/tscTask.ts

```diff
@@ -133,7 +133,7 @@ export class TscTask extends LeafTask {
     private remapSrcDeclFile(fullPath: string) {
         if (!this._sourceStats) {
             const config = this.readTsConfig();
-            this._sourceStats = config ? config.fileNames.map(fs.lstatSync) : [];
+            this._sourceStats = config ? config.fileNames.map(v => fs.lstatSync(v)) : [];
```

Comment:
do you know what the effect is here when running in node 14 without this change?  i guess either way the build-tools changes will probably want to be in a separate change, get published, then this change bumps the build-tools versions instead